### PR TITLE
update readme for removing repeated content

### PR DESCRIPTION
**Pull request by thinkerou**
# Description

removing repeated content on readme.md

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation
* [ ] Specification has been updated
* [ ] Provided sample for the feature

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Dapr - Any language, any framework, anywhere
+# Dapr - any language, any framework, anywhere
```
Comment (amanbha) [inline README.md]: This was intentional, this should not be changed.

File: README.md [hunk 2]
```diff
@@ -7,10 +7,6 @@ Dapr is a portable, event-driven, runtime for building distributed applications
 [![Gitter](https://badges.gitter.im/Dapr/community.svg)](https://gitter.im/Dapr/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
-- [dapr.io](https://dapr.io)
```
Comment (amanbha) [inline README.md]: This was intentional, this should not be removed.

File: README.md (hunk 1 above)
Comment (thinkerou) [reply README.md]: thanks! revert.

File: README.md (hunk 2 above)
Comment (thinkerou) [reply README.md]: thank you! I suggest that add the url to the description of repo, ok?

Comment (amanbha) [reply README.md]: Yes, that would be great.

**Pull request merged**